src/sql/src/names.rs

```diff
@@ -488,6 +493,10 @@ impl AstDisplay for ResolvedItemName {
                 }
             }
             ResolvedItemName::Cte { name, .. } => f.write_node(&Ident::new_unchecked(name)),
+            ResolvedItemName::ContinualTask { name, .. } => {
+                // WIP
+                f.write_node(&Ident::new_unchecked(&name.item))
```

Comment:
You should be able to call `f.write_str` on the whole `name` here, [`write_str`](https://github.com/MaterializeInc/materialize/blob/main/src/sql-parser/src/ast/display.rs#L101-L103) takes anything that impls `fmt::Display`

Reply:
Thanks! I had a sense it'd be faster just to ask you on this :)